compiler/src/parsing/lexer.re

```diff
@@ -214,6 +214,8 @@ let rec token = lexbuf => {
   | (unsigned_float, 'w') => positioned(WASMF32(sub_lexeme(lexbuf, 0, -1)))
   | (unsigned_float, 'W') => positioned(WASMF64(sub_lexeme(lexbuf, 0, -1)))
   | (unsigned_int, 't') => positioned(BIGINT(sub_lexeme(lexbuf, 0, -1)))
+  | (unsigned_int, '/', unsigned_int, 'r') =>
```

Comment:
I almost wish this was `'q'` instead of `'r'`, but I think that'd make people cry...